dapr/proto/runtime/v1/appcallback.proto

```diff
@@ -143,46 +143,7 @@ message BindingEventResponse {
 // ListTopicSubscriptionsResponse is the message including the list of the subscribing topics.
 message ListTopicSubscriptionsResponse {
   // The list of topics.
-  repeated TopicSubscription subscriptions = 1;
-}
-
-// TopicSubscription represents topic and metadata.
-message TopicSubscription {
-  // Required. The name of the pubsub containing the topic below to subscribe to.
-  string pubsub_name = 1;
-
-  // Required. The name of topic which will be subscribed
-  string topic = 2;
-
-  // The optional properties used for this topic's subscription e.g. session id
-  map<string,string> metadata = 3;
-
-  // The optional routing rules to match against. In the gRPC interface, OnTopicEvent
-  // is still invoked but the matching path is sent in the TopicEventRequest.
-  TopicRoutes routes = 5;
-
-  // The optional dead letter queue for this topic to send events to.
-  string dead_letter_topic = 6;
-}
-
-message TopicRoutes {
-  // The list of rules for this topic.
-  repeated TopicRule rules = 1;
-
-  // The default path for this topic.
-  string default = 2;
-}
-
-message TopicRule {
-  // The optional CEL expression used to match the event.
-	// If the match is not specified, then the route is considered
-	// the default.
-  string match = 1;
-
-  // The path used to identify matches for this subscription.
-  // This value is passed in TopicEventRequest and used by OnTopicEvent to "switch"
-  // inside the handler.
-  string path = 2;
+  repeated common.v1.TopicSubscription subscriptions = 1;
```

Comment:
It's to support #814. Eventually there'll be a gRPC method that allows subscribing to topics on-demand. Because that will be in the `Dapr` service (in dapr.proto), I alrady moved the message definition inside `common.proto`